docs/adr/0007-alerting-setup.md

```diff
@@ -0,0 +1,49 @@
+# Alerting setup for Operate First monitoring
+
+* Status: accepted
+* Deciders: hemajv, 4n4nd, anishasthana, HumairAK, tumido, mhild
+
+Technical Story: [issue-1](https://github.com/operate-first/SRE/issues/14), [issue-2](https://github.com/operate-first/SRE/issues/19), [issue-3](https://github.com/operate-first/blueprint/issues/16)
+
+## Context and Problem Statement
+
+As we have multiple services deployed and monitored in the Operate First environment (ex. Jupyterhub, Argo, Superset, Observatorium, etc), we need to implement an incident reporting setup for handling outages/incidents related to the services.
+
+All the services are being monitored by [Prometheus](https://prometheus.io/). Prometheus scrapes and stores time series data identified by metric key/value pairs for each of the available services. These metrics can be used for measuring the service performance and alerting on any possible service degradation such as basic availability, latency, durability etc.
+
+Alerting with Prometheus is separated into two parts. Alerting rules in Prometheus servers send alerts to an [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). The Alertmanager then manages those alerts, including silencing, inhibition, aggregation and sending out notifications via methods such as email, on-call notification systems, and chat platforms.
+
+There are multiple on-call notification systems such as [PagerDuty](https://www.pagerduty.com/), [JIRA](https://www.atlassian.com/software/jira) etc that can be used for incident reporting, but which of these tools are best suitable for reporting the outages/incidents to our users?
+
+## Decision Drivers <!-- optional -->
+
+* Visibility of incident reporting
+  * How can the incidents be tracked and reported in an open and transparent manner for our users?
+* Compatibility with Prometheus
+  * Is the incident reporting tool compatible with Prometheus i.e. can it handle/receive Prometheus alerts?
+* Complexity and cost of incident reporting tool
+  * How easy/hard is it to manage and operate the incident reporting tool?
+  * Is it a free or paid tool?
+
```

Comment:
```suggestion
  * Is it opensouce?

```

This is a big factor, at least for me, since we aim to do all of this OperateFirst in the open and we should be faithful to open source in the tooling we use.

Reply:
added :+1: